pkg/apis/kops/v1alpha2/cluster.go

```diff
@@ -65,6 +65,9 @@ type ClusterSpec struct {
 	// MasterInternalName is the internal DNS name for the master nodes
 	MasterInternalName string `json:"masterInternalName,omitempty"`
 
+	// AdditionalSANs adds additional Subject Alternate Names to apiserver cert that kops generates
+	AdditionalSANs []string `json:"additionalSans,omitempty"`
```

Comment:
why not `additionalSAN`?  The capitalization is bit odd given there's `networkCIDR`.